src/streaming/StreamProcessor.js

```diff
@@ -114,18 +113,14 @@ function StreamProcessor(config) {
             requestModifier: RequestModifier(context).getInstance()
         });
 
-        representationController = RepresentationController(context).create();
-        representationController.initialize(this);
-
         fragmentModel = scheduleController.getFragmentModel();
         fragmentModel.setLoader(fragmentLoader);
+
+        representationController = RepresentationController(context).create();
+        representationController.initialize(this);
     }
 
     function reset(errored) {
-        if (fragmentModel) {
-            fragmentModel.reset();
-            fragmentModel = null;
-        }
```

Comment:
Gotcha fair enough. I saw the `.create` for `RepresentationController` above and below this and assume we were creating `FragmentModel`... but we're not. So disregard